apps/web-giddh/src/app/services/country-code.service.ts

```
@@ -0,0 +1,135 @@
+    /** Default country code to use when API calls fail */
+    private readonly DEFAULT_COUNTRY_CODE = 'in';
+    /** API endpoint to get client's IP address */
+    private readonly MOBILE_NUMBER_SELF_URL = 'https://api.db-ip.com/v2/free/self';
+    /** Primary API endpoint to get country code from IP address */
+    private readonly MOBILE_NUMBER_IP_ADDRESS_URL = 'http://ip-api.com/json/';
+    /** Fallback API endpoint to get country code from IP address */
+    private readonly MOBILE_NUMBER_ADDRESS_JSON_URL = 'https://ipinfo.io/';
+
+    constructor() {
+        if (CountryCodeService.instance) {
+            return CountryCodeService.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)